CONTRIBUTING.md

```diff
@@ -154,6 +154,7 @@ AKO can be tested from your laptop. Please follow the instructions:
         export CLOUD_NAME=<Avi-cloud-name>
         export CLUSTER_NAME=<your-unique-cluster-name>
         export NODE_NETWORK_LIST='[{"networkName":"Nw1","cidrs":["10.79.168.0/22"]}]'
+        export SEG_NAME="Default-Group"
```

Comment:
Maybe we want to specify in what conditions the "Default-Group" has to be specified?
For advancedL4 if this value isn't set, then the code would assume "Default-Group".
If you specify the "Default-Group" here, it seems like the default use case, but users can select a non-default SE group as well. So maybe I will just alter it and say:
export SEG_NAME=<SE Group name>